includes/discovery/processors/cambium.inc.php

```diff
@@ -21,7 +21,7 @@
             '0',
             'cambium-cpu',
             $descr,
-            '100',
+            '10',
```

Comment:
This looks good to me, option 1 seems like the best way to go here. I've noticed ePMP haven't been reporting right but was to busy to look into. Happy to see a fix.